features/step_definitions/task_steps.rb

```diff
@@ -10,7 +10,16 @@
 
 Given /the following Iteration exist:$/ do |table|
   table.hashes.each do |hash|
-    Iteration.create hash
+    iter = Iteration.create hash
+    iter.set_timestamp("2012/2/4", "2022/2/4")
+  end
+  @iteration = Iteration.all
+end
+
+Given /the following Iterations exist start "([^\"]*)" end "([^\"]*)":$/ do |startime, endtime, table|
```

Comment:
Is there a reason you want to divide the step of creating an iteration from setting timestamps for that iteration?

Reply:
I thought in one of the previous request, the iteration that is not template must have starting date and end date. So I set it for every iteration.